tests/html/HTMLImports.html

```diff
@@ -1,51 +0,0 @@
-<!doctype html>
```

Comment:
Removed because duplicate of dynamic.html tests, was testing internal properties.